pkg/sql/generator.go

```diff
@@ -62,17 +58,10 @@ func (p *planner) makeGenerator(ctx context.Context, t *parser.FuncExpr) (planNo
 		return nil, errors.Errorf("FROM expression is not a generator: %s", t)
 	}
 
-	var columns sqlbase.ResultColumns
-	if len(tType.Cols) == 1 {
-		columns = sqlbase.ResultColumns{sqlbase.ResultColumn{Name: origName, Typ: tType.Cols[0]}}
-	} else {
-		columns = make(sqlbase.ResultColumns, len(tType.Cols))
-		for i, t := range tType.Cols {
-			columns[i] = sqlbase.ResultColumn{
-				Name: fmt.Sprintf("column%d", i+1),
-				Typ:  t,
-			}
-		}
+	columns := make(sqlbase.ResultColumns, len(tType.Cols))
+	for i := range columns {
```

Comment:
I just realized PG lets you specify the label for a single-column generator:
```
select * from unnest(array[1,2,3]) a;
+--------+
| unnest |
+--------+
|      1 |
|      2 |
|      3 |
+--------+
(3 rows)
```
is that something we can easily support?